realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -300,7 +300,9 @@ protected void removeAllListeners() {
     }
 
     /**
-     * Writes a compacted copy of the Realm to the given destination File.
+     * Writes a compacted copy of the Realm to the given destination File. On a synced Realm it will
+     * write a copy stripping the client file identifier that will allow to reuse the Realm without
```

Comment:
`without redownloading the realm` sounds a bit funny...maybe rephrase to `resue the Realm on another device`?

Reply:
Maybe:
`Writes a compacted copy of the Realm to the given destination File. The resulting file can be used as initial dataset to bootstrap local or synced Realm in other devices.`